app/routes/goal_routes.py

```diff
@@ -1 +1,68 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, request, make_response, Response
+from app.models.goal import Goal
+from app.models.task import Task
+from .. import db
+from .route_utilities import validate_model, create_model, get_models_with_filters
+
+bp = Blueprint("goals_bp", __name__, url_prefix="/goals")
+
+@bp.post("")
+def create_goal():
+    request_body = request.get_json()
+    return create_model(Goal,request_body)
+
+@bp.post("/<goal_id>/tasks")
+def add_tasks_to_goal(goal_id):
+    goal = validate_model(Goal, goal_id)
+    request_body = request.get_json()
+    task_ids = request_body.get("task_ids", [])
+    goal.tasks = []
+
+    for task_id in task_ids:
+        task = validate_model(Task, task_id)
+        goal.tasks.append(task)
+
+    db.session.commit()
+
+    return {"id": goal.id, "task_ids": task_ids}, 200
+
+@bp.get("/<goal_id>/tasks")
+def get_tasks_of_goal(goal_id):
+    goal = validate_model(Goal, goal_id)
+    tasks = [task.to_dict() for task in goal.tasks]
+
+    return {"id": goal.id, "title": goal.title, "tasks": tasks}, 200
```

Comment:
```suggestion
    return {"id": goal.id, "title": goal.title, "tasks": tasks}
```

Default status code sent back to client 200 so we can omit it (like you do on line 45 below).

Reply:
Fixed as requested.